### DISCUSSION TITLE
ScrolledWindow Example

### DISCUSSION THREAD

**Discussion by lildinti** [wxFormBuilder/wxFormBuilder#745] [General]
Hello there!
Any chance to get an example of scrolledwindow widget in an fbp file? 
Thanks
Lilidinti

**Comment by sodevel** [comment]
I'm afraid i don't understand the question. Do you want to see the XML representation of wxScrolledWindow inside the wxFormBuilder project file? Create a new Project, add a Top Level element, add a Sizer, add the wxScrolledWindow, save the Project and open the Project File with a text editor.
